authentication/authentication-styles-2.css

```diff
@@ -0,0 +1,110 @@
+body {
+    width: 100vw;
+    height: 100vh;
+    background-color: #EEEEEE;
+
+    padding: 20px;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.card {
+    max-width: 50%;
+    background-color: white;
+    display: flex;
+    flex-direction: row;
+
+    border-radius: 16px;
+    box-shadow: 4px 4px 8px grey;
+
+}
+
+a {
+    color: lightblue;
+}
+
+.form {
+    flex: 1;
+
+    display: flex;
+    flex-direction: column;
+    padding-left: 32px;
+    padding-right: 32px;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    justify-content: space-between;
+    gap: 8px;
+}
+
+.create-account-btn {
+    background-color: rgb(255, 166, 0);
+    font-weight: bold;
+    box-shadow: none;
+    border: 0;
+    padding-top: 12px;
+    padding-bottom: 12px;
+    border-radius: 8px;
+    font-size: medium;
+}
+
+.form-header { 
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+
+    gap: 16px;
+}
+
+
+.form-input { 
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    gap: 12px;
+
+    padding: 16px;
+}
+
+.form-input > input {
+    align-self: center;
+    border-radius: 4px;
+    padding: 12px 16px;
+    box-sizing: content-box;
+    width: 100%;
+}
+
+p {
```

Comment:
Same as above, we should not style the whole p element. Instead we can use classname / id